icon docs (#1661)
NuGet/docs.microsoft.com-nuget: docs/reference/errors-and-warnings/NU5047.md

---
title: NuGet Error NU5047
description: NU5047 Error code
author: dominoFire
ms.author: feaguila
ms.date: 9/4/2019
ms.topic: reference
ms.reviewer: karann
f1_keywords: 
  - NU5047
---

# NuGet Error NU5047

<pre>The icon file size must not exceed 1 megabyte.</pre>

# Issue 

> The file that is specified as the package icon has a filesize larger than 1 megabyte (MB). NuGet only allow icons whose filesize is less than 1 MB.

Reviewer: ```suggestion
The file that is specified as the package icon is larger than 1 megabyte (MB). NuGet only allows icons whose file size is less than 1 MB.
```

# Solution

> Use an image editor program to edit and downsize the package icon file.

Reviewer: ```suggestion
Use an image editor program to reduce the size of the package icon file.
```
